### PR TITLE
www: prettify XCP-ng-index.html deploy.js

### PR DESCRIPTION
**Pull request by arnaudgs**
Prettify with prettier the XCP-ng-index.html and deploy.js files before made changes inside the code of thoses files.

### CODE REVIEW

File: src/xenserver/opt/xensource/www/XCP-ng-index.html [hunk 1]
```diff
@@ -1,604 +1,555 @@
...
+    }
+
+    button {
+      opacity: 0.9;
+      text-decoration: none;
+      color: #fff;
+      border-radius: 4px;
+      text-transform: uppercase;
+      background-color: #cc584c;
+      /* xcp-ng red */
```
Comment (stormi) [inline src/xenserver/opt/xensource/www/XCP-ng-index.html:84]: Many comments are misplaced by the prettifier: they're placed after the line they initially were on, and should instead be placed before.

Comment (stormi) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html:84]: One comment is still misplaced.

File: src/xenserver/opt/xensource/www/XCP-ng-index.html [hunk 2]
```diff
@@ -316,7 +316,7 @@ <h3 class="subtitle">Management tools</h3>
         <div class="col-md">
           <div class="box">
             <h4>Xen Orchestra</h4>
-            <a href="#ex1" rel="modal:open"><img class="preview" src="./asset/xoa.jpg" /></a>
+            <a href="#ex1" rel="modal:open"><img class="preview" src="./asset/xoa.jpg" alt="Open XOA" /></a>
```
Comment (stormi) [inline src/xenserver/opt/xensource/www/XCP-ng-index.html]: This does not look related to this PR and is not mentioned in any commit messages.

**Review by stormi — CHANGES_REQUESTED** [review]

File: src/xenserver/opt/xensource/www/XCP-ng-index.html (hunk 2 above)
Comment (arnaudgs) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html]: done

File: src/xenserver/opt/xensource/www/XCP-ng-index.html (hunk 1 above)
Comment (arnaudgs) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html:84]: done

Comment (arnaudgs) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html:84]: They are also useless comments :D I should remove them next time

File: src/xenserver/opt/xensource/www/XCP-ng-index.html [hunk 3]
```diff
@@ -1,604 +1,555 @@
...
+    }
+
+    /* Tooltip text */
+    .tooltip .tooltiptext {
+      visibility: hidden;
+      width: 120px;
+      top: 100%;
+      left: 50%;
+      margin-left: -60px;
+      /* Use half of the width (120/2 = 60), to center the tooltip */
```
Comment (stormi) [inline src/xenserver/opt/xensource/www/XCP-ng-index.html:221]: Still misplaced

Comment (arnaudgs) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html:221]: Je vais supprimer les commentaires

Comment (arnaudgs) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html:221]: done

Comment (stormi) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html:221]: The prettify operation should be neutral, so I would like you to restore the comment.

Comment (arnogues) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html]: but after I will kill all the bad stuff in the code. I think I spend too much time on this useless stuff

Comment (arnaudgs) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html:221]: I rename the commit to : remove useless comments ?

Comment (stormi) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html:221]: No, the commit should only prettify the code and leave the comments alone

**Comment by stormi** [comment]
So, I can see you're upset that there are so many iterations on this very simple PR (I received the mail notification for the comment that was deleted), and to be honest, so am I.

So let's try to redefine clearly the boundaries of this PR, and make some implicit assumptions explicit, to move forward.

Prettifying the code is not a requirement for us on this web page. Our real need is replacing icons and traceability of the new icons (origin and license). We do require that new code added comes with at least some level of consistency with the existing, and that's why there had been a comment in the previous PR regarding the indentation of the added code, following which you decided to reformat the whole files rather than just fixing the added code.

When ~~you~~ we (update: I remember I suggested it, due to the rebase issues with the formatting commit in the middle of the PR) decided to restart the PR from scratch, you suggested we start with a simple PR where you would just run `pretty` on the existing code, before doing other changes. I agreed, not because we need the reformatting, but because you prefer to work this way, so that you can handle the rest of the task in a way that fits you.

However, the expected scope of this pull request is just this: reformatting with a tool. This could have been done in just one iteration: run the tool, check the diff, notice that the tool took wrong decisions for 4 comments (no tool is perfect), move the 4 comments one line above, commit, push, open pull request.

Adding an alt text to an object, removing a comment, it's all out of scope for this reformatting PR. It was not obvious, I did not think of stating it explicitly before you created the PR, because I thought it was a commonly accepted rule: never mix a reformatting commit with other changes (except changes that fix mistakes the reformatting tool made). This was my mistake.

So please update this PR so that the only changes are the reformatting by `pretty`, with the 4 comments it misplaced placed one line higher. 

Then we'll take the time to discuss the next steps and make sure the objectives and proposed solution are clear for both of us.

File: src/xenserver/opt/xensource/www/XCP-ng-index.html (hunk 3 above)
Comment (arnaudgs) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html:221]: then done

**Comment by arnaudgs** [comment]
Hello I completlly understand. 
Prettifying the page is a requirement for me, because it clean everything : space, tabs, bad spaces, and it's better for working and coding after.

Then don't worry for the next steps, they will be better, I have already prepared the next stuff (also font icons and source and licence are ready).

I think it was because for each change I squash the others commits but I didn't have the good process to squash commits. 
Now I have a good process and prepared for next step.

**Review by stormi — APPROVED** [review]
